ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -416,3 +416,33 @@ ethernet_interfaces:
         source_interface: Loopback0
         vrf: MGMT
       - ip_helper: 10.10.64.151
+
```

Comment:
Please move these changes to the `eos_cli_config_gen` scenario instead of 4.0. It will solve the conflict at the same time :)